shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java

```diff
@@ -86,6 +93,16 @@ protected void dispatchDraw(Canvas canvas) {
         }
     }
 
+    @Override
+    public void setVisibility(int visibility) {
+        super.setVisibility(visibility);
+        if (visibility == VISIBLE) {
```

Comment:
how about the user sets the Visibility to Visible but didn't set autoStart = true; wouldn't that cause the animation to be stopped?